webviews/src/BranchBasePicker.tsx

```
@@ -0,0 +1,528 @@
+          {/* oxlint-disable-next-line jsx-a11y/prefer-tag-over-role */}
+          <div id={listboxId} className="base-picker-list" role="listbox" aria-label={label("branchPickerOpen")}>
+            {loadState === "loading" ? (
+              <div className="base-picker-status">{label("branchPickerLoading")}</div>
+            ) : loadState === "error" ? (
+              <div className="base-picker-status base-picker-status-error">{label("branchPickerLoadFailed")}</div>
+            ) : flat.length === 0 ? (
+              <div className="base-picker-status">{label("branchPickerNoMatches")}</div>
+            ) : (
+              flat.map((entry, index) => (
+                <BranchPickerRowView
+                  key={`${entry.groupId}:${entry.row.ref}:${index}`}
```

Biome (2.5.0)

[error] Avoid using the index of an array as key property in an element.

(lint/suspicious/noArrayIndexKey)